guardrails_api/utils/has_internet_connection.py

```diff
@@ -0,0 +1,9 @@
+import requests
+
+
+def has_internet_connection() -> bool:
+    try:
+        requests.get("https://www.guardrailsai.com/")
```

Comment:
I think you might have to do:
```suggestion
        res = requests.get("https://www.guardrailsai.com/")
        res.raise_for_status()
```
I believe to get it to fail on a bad http status code

Reply:
Good call, thank you!